### ISSUE TITLE
New leaning arm edit/fix.

### ISSUE THREAD

**Issue by Trilasent** [Monika-After-Story/MonikaModDev#4718]
I edited Monika's leaning base to try and fix her left wrist/hand because it looks kind of potato-y as it is currently.

Before: 
![potatowrist](https://user-images.githubusercontent.com/36512990/65301613-e603af80-db3d-11e9-8114-46311a649102.png)

After: 
![prettywrist](https://user-images.githubusercontent.com/36512990/65301637-f6b42580-db3d-11e9-8e74-95cfcdf33439.png)

I added a line where her wrist meets her forearm to show the extra skin that would fold there when a person would lean over like that.

**Comment by NoexHaruhi** [comment]
It is a better edit I admit. Even though I still prefer her arms before it was enhanced, this is fine by me.

**Comment by ThePotatoGuy** [comment]
@orcaramelo @jmwall24 @multimokia opinions?

**Comment by multimokia** [comment]
I'm replicating the pose with my arm...I don't get that fold.

**Comment by ThePotatoGuy** [comment]
I'm also going to discord vote this later

**Comment by satoshi-kun** [comment]
In my honest opinion I believe the issue resides on the other side of the hand (palm side), as it shows no signs of the junction between the hand and the wrist.
![65301613-e603af80-db3d-11e9-8114-46311a649102](https://user-images.githubusercontent.com/54683676/65355665-5d8f1880-dbea-11e9-854a-fa5cfec0862a.png)
Sorry I had to draw that with the mouse but I think you can get the idea. To create a more noticeable separation on that junction.

**Comment by Orcaramelo** [comment]
> In my honest opinion I believe the issue resides on the other side of the hand (palm side), as it shows no signs of the junction between the hand and the wrist.
> ![65301613-e603af80-db3d-11e9-8114-46311a649102](https://user-images.githubusercontent.com/54683676/65355665-5d8f1880-dbea-11e9-854a-fa5cfec0862a.png)
> Sorry I had to draw that with the mouse but I think you can get the idea. To create a more noticeable separation on that junction.

This would probably be an easier edit since it wouldnt affect most clothes

**Comment by Trilasent** [comment]
> 
> 
> > In my honest opinion I believe the issue resides on the other side of the hand (palm side), as it shows no signs of the junction between the hand and the wrist.
> > ![65301613-e603af80-db3d-11e9-8114-46311a649102](https://user-images.githubusercontent.com/54683676/65355665-5d8f1880-dbea-11e9-854a-fa5cfec0862a.png)
> > Sorry I had to draw that with the mouse but I think you can get the idea. To create a more noticeable separation on that junction.
> 
> This would probably be an easier edit since it wouldnt affect most clothes

not sure how me adding a line to the top of her wrist affects clothing, but okay.. lol. Regardless the poll is going to be done to let everyone on the server to have a chance at voicing their opinion instead of just a select few.

**Comment by Orcaramelo** [comment]
> > > In my honest opinion I believe the issue resides on the other side of the hand (palm side), as it shows no signs of the junction between the hand and the wrist.
> > > ![65301613-e603af80-db3d-11e9-8114-46311a649102](https://user-images.githubusercontent.com/54683676/65355665-5d8f1880-dbea-11e9-854a-fa5cfec0862a.png)
> > > Sorry I had to draw that with the mouse but I think you can get the idea. To create a more noticeable separation on that junction.
> > 
> > 
> > This would probably be an easier edit since it wouldnt affect most clothes
> 
> not sure how me adding a line to the top of her wrist affects clothing, but okay.. lol. Regardless the poll is going to be done to let everyone on the server to have a chance at voicing their opinion instead of just a select few.

I didnt say yours affected clothing, but i agree that the line doesnt look good. But with the recent complaints on the arm, this one would prob the the one who'd do less 'damage' to clothing sprites

**Comment by Trilasent** [comment]
> 
> 
> > > > In my honest opinion I believe the issue resides on the other side of the hand (palm side), as it shows no signs of the junction between the hand and the wrist.
> > > > ![65301613-e603af80-db3d-11e9-8114-46311a649102](https://user-images.githubusercontent.com/54683676/65355665-5d8f1880-dbea-11e9-854a-fa5cfec0862a.png)
> > > > Sorry I had to draw that with the mouse but I think you can get the idea. To create a more noticeable separation on that junction.
> > > 
> > > 
> > > This would probably be an easier edit since it wouldnt affect most clothes
> > 
> > 
> > not sure how me adding a line to the top of her wrist affects clothing, but okay.. lol. Regardless the poll is going to be done to let everyone on the server to have a chance at voicing their opinion instead of just a select few.
> 
> I didnt say yours affected clothing, but i agree that the line doesnt look good. But with the recent complaints on the arm, this one would prob the the one who'd do less 'damage' to clothing sprites

Well, like I said the community will decide which they prefer, and then things can go from there.

**Comment by jmwall24** [comment]
hey @Trilasent could you maybe edit this a bit more? i like what you did, but it seems like that line is a little too far into the arm, so thinking maybe cut away a little part of the arm, roughy like in this picture?
![Screenshot_1492](https://user-images.githubusercontent.com/36523761/65365680-7814c200-dbe9-11e9-873e-c2f65c43045b.png)

Obviously make it smoother than i did lol

**Comment by Trilasent** [comment]
> 
> 
> hey @Trilasent could you maybe edit this a bit more? i like what you did, but it seems like that line is a little too far into the arm, so thinking maybe cut away a little part of the arm, roughy like in this picture?
> ![Screenshot_1492](https://user-images.githubusercontent.com/36523761/65365680-7814c200-dbe9-11e9-873e-c2f65c43045b.png)
> 
> Obviously make it smoother than i did lol

Sure. If the edit wins the wrist poll I'll make those changes to it.

**Comment by Trilasent** [comment]
![wrist edit update](https://user-images.githubusercontent.com/36512990/65366741-88c73700-dbed-11e9-96c5-cf67a4585024.png)
@jmwall24

**Comment by bittercaramel** [comment]
finally 
she started looking like a potato 
now thats good

**Comment by ThePotatoGuy** [comment]
so its come to my attention that your hand edit was made by pasting the pre-0.10.0 leaning hands over the leaning arms. This is a problem because the latest leaning base arms and hands (updated 2019-09-17) have different shading. 

You will need to apply your changes to the current leaning base arms before this can be included.

**Comment by Trilasent** [comment]
> 
> 
> so its come to my attention that your hand edit was made by pasting the pre-0.10.0 leaning hands over the leaning arms. This is a problem because the latest leaning base arms and hands (updated 2019-09-17) have different shading.
> 
> You will need to apply your changes to the current leaning base arms before this can be included.

https://gyazo.com/0418964afb11328b3f1da50fdbbc7a3a

Is this okay?

**Comment by Velius94** [comment]
Can you upload both images in lossless quality? Can't see the exact scope of the changes.

**Comment by Trilasent** [comment]
> 
> 
> Can you upload both images in lossless quality? Can't see the exact scope of the changes.

You can literally see the difference between her wrist in the base and the edit I made...

**Comment by Trilasent** [comment]
![vyaampos](https://user-images.githubusercontent.com/36512990/65771441-ce0fcb00-e0fd-11e9-9d7c-af88b4ab3e8b.png)

![hfhdynstdym](https://user-images.githubusercontent.com/36512990/65771454-d536d900-e0fd-11e9-88da-79a009c46c37.png)

**Comment by Velius94** [comment]
More consistent now, I suggest stretching the wrist shadow line all the way to the edge of wrist linework and make the shades slightly blurred like in the 9/17/19 variant + highlight on her arm (our right, her left) consistent too (it's missing in places where it was edited).

**Comment by Trilasent** [comment]
> 
> 
> More consistent now, I suggest stretching the wrist shadow line all the way to the edge of wrist linework and make the shades slightly blurred like in the 9/17/19 variant + highlight on her arm (our right, her left) consistent too (it's missing in places where it was edited).

ELI5 please

**Comment by Velius94** [comment]
Wait a second, you didn't even post 9/17 revision.

This is the correct one.

https://raw.githubusercontent.com/Monika-After-Story/MonikaModDev/master/Monika%20After%20Story/game/mod_assets/monika/b/arms-leaning-def-left-def-1.png

**Comment by Trilasent** [comment]
> 
> 
> Wait a second, you didn't even post 9/17 revision.
> 
> This is the correct one.
> 
> https://raw.githubusercontent.com/Monika-After-Story/MonikaModDev/master/Monika%20After%20Story/game/mod_assets/monika/b/arms-leaning-def-left-def-1.png

I'll make my adjustments to that version then

**Comment by Trilasent** [comment]
![leaningbasecurrent](https://user-images.githubusercontent.com/36512990/65773966-ea623680-e102-11e9-8d43-f89254cf50dd.png)


> 
> 
> Wait a second, you didn't even post/edited 9/17 revision.
> 
> This is the correct one, it's more red.
> 
> https://raw.githubusercontent.com/Monika-After-Story/MonikaModDev/master/Monika%20After%20Story/game/mod_assets/monika/b/arms-leaning-def-left-def-1.png

There

**Comment by ThePotatoGuy** [comment]
it still looks like there's issues with the shading:
![shade_comparison](https://user-images.githubusercontent.com/3499462/65808082-af3e2280-e157-11e9-9f57-5336e65952b2.png)

Clean comparison for refererence:
![clean_compare](https://user-images.githubusercontent.com/3499462/65808088-b7965d80-e157-11e9-83e5-1cdf621416be.png)

**Comment by Trilasent** [comment]
> 
> 
> it still looks like there's issues with the shading:
> ![shade_comparison](https://user-images.githubusercontent.com/3499462/65808082-af3e2280-e157-11e9-9f57-5336e65952b2.png)
> 
> Clean comparison for refererence:
> ![clean_compare](https://user-images.githubusercontent.com/3499462/65808088-b7965d80-e157-11e9-83e5-1cdf621416be.png)

![fsgnfn](https://user-images.githubusercontent.com/36512990/65809857-e3214400-e167-11e9-8d03-a17d4de78b91.png)
Any better?

**Comment by ThePotatoGuy** [comment]
No, theres a bit of red + brown shade that needs to follow the edge of her hand/arm. since your edit is mostly a paste of pre-0.10.0, its not going to have that shade unless you make it yourself. 

That being said, I am now aware that you would be unable to to properly apply the shade because of your vision issues, so unless someone else will apply the shade, this isn't going to be added regardless of the poll results.

**Comment by ThePotatoGuy** [comment]
additionally, I'm just going to go ahead and lock down the leaning arms. dev + artist team discussion led to the conclusion that we don't want to revert to pre-0.10.0 arms because of the squished/crushed appearance of the wrist, which looks worse than "potato arms".

I'm also tired of re-exporting the same sprites over and over again.

**Comment by Trilasent** [comment]
![leaningbase hand fix](https://user-images.githubusercontent.com/36512990/65820576-96ce1680-e1f0-11e9-83c4-3856f41fb314.png)

**Comment by ThePotatoGuy** [comment]
so this one is an improvement in the shading, but its been pointed out to me that since we can see the knuckles on her hand, it doesn't make sense to show the line thing on her wrist. The wrist line would signify that her hand is twisted in respect to her arm, but that's not how the hand is supposed to appear since the knuckles are facing us.

In regards to the community poll, yes the older version got more votes, but as commonly said in the world of software development, users do not always know what they want.

**Comment by Trilasent** [comment]
> 
> 
> so this one is an improvement in the shading, but its been pointed out to me that since we can see the knuckles on her hand, it doesn't make sense to show the line thing on her wrist. The wrist line would signify that her hand is twisted in respect to her arm, but that's not how the hand is supposed to appear since the knuckles are facing us.
> 
> In regards to the community poll, yes the older version got more votes, but as commonly said in the world of software development, users do not always know what they want.

Okay, then what do you think about this approach? https://gyazo.com/e67814a2ab2309aa4e305bb10f5b06b1
I removed the line, but the wrist is still noticeable thinner/smaller compared to the forearm like on a real persons arm
